v3/plugins/redhat/vscode-openshift-connector/0.1.5/meta.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-openshift-connector
+version: 0.1.5
+type: VS Code extension
+displayName: OpenShift Connector
+title: OpenShift Connector
+description: Interacting with Red Hat OpenShift clusters and providing a streamlined developer experience using Eclipse Che
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-openshift-tools
+category: Other
+firstPublicationDate: "2020-05-20"
+spec:
+  containers:
+    - image: "quay.io/eclipse/che-sidecar-openshift-connector:0.1.5-f1b9673"
+      name: "vscode-openshift-connector"
+      memoryLimit: "2G"
```

Comment:
`2G` means this version of openshift-connector quadruples the memory requirement from previous; is there not a tighter bound? Maybe `768Mi`?

Reply:
It was an experiment to see if the plugin memory limit was an issue (it was), I'll find a tighter bound. It's worth noting though that this version of the plugins uses OpenShift 4, which may have higher memory requirements.

Reply:
Looks like 1500Mi is the sweet spot, 1G or less and the container won't start due to OOM errors.